geneocr-ui/1.0/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# base image
+FROM node:13.12.0-alpine as build
+
+# metadata
+LABEL base_image="node:13.12.0-alpine"
+LABEL version="1"
+LABEL software="GeneOCR UI"
+LABEL software.version="1.0.0"
+LABEL about.summary="UI of geneocr tool, which reads gene names in images"
+LABEL about.home="https://github.com/bedapub/geneocr-ui"
+LABEL maintainer="Jitao David Zhang"
+LABEL maintainer.email="jitao_david.zhang@roche.com"
+
+# install dependencies
+RUN export DEBIAN_FRONTEND=noninteractive && \
```

Comment:
apt-get does not work on alpine

Reply:
I took your advise and tested it with docker locally, I will push a working version now